charts/mobu/values.yaml

```diff
@@ -4,6 +4,13 @@
 # Each flock listed will be automatically started when mobu is started.
 autostart: []
 
+# -- Cachemachine image policy.  Must be one of "desired" or
+# 'available'.  Determines whether cachemachine reports the images it
```

Comment:
Please use double quotes (`"available"` instead of `'available'`). It matches `"desired"` and the usage below.

Reply:
Done